catalog/transformer/catalog-transformer-pptx/src/main/java/ddf/catalog/transformer/input/pptx/PptxInputTransformer.java

```diff
@@ -150,7 +150,8 @@ private Metacard extractInitialMetadata(InputStream input)
   private void extractThumbnail(Metacard metacard, InputStream input) throws IOException {
 
     ClassLoader originalContextClassLoader = Thread.currentThread().getContextClassLoader();
-    Thread.currentThread().setContextClassLoader(getClass().getClassLoader());
+    // This class is arbitrary. It just needs to be part of the bundle exporting POI classes
+    Thread.currentThread().setContextClassLoader(SlideShowFactory.class.getClassLoader());
```

Comment:
✏️ Add an itest for ingesting a powerpoint and verifying that the thumbnail metacard attribute is populated to prevent this regression in the future.

Reply:
Double check that the iTest you add fails without this change.

Reply:
Credit to @emmberk for writing the itest. Thank you!